web-js/src/components/WelcomePage.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import * as Realm from "realm-web";
+import {
+  Container,
+  TextField,
+  Button,
+  IconButton,
+  Card,
+  Typography,
+  InputAdornment,
+} from "@material-ui/core";
+import { useRealmApp } from "./RealmApp";
+import { MoreInfoTemplateAndDocs } from './MoreInfo'
+import Visibility from "@material-ui/icons/Visibility";
+import VisibilityOff from "@material-ui/icons/VisibilityOff";
+import { toggleBoolean } from "../utils";
```

Comment:
I suggest separating imports of external packages vs relative files into two sections (external first) to improve readability here.